app/views/command-line.html

```diff
@@ -62,6 +62,20 @@ <h1 id="cli">Command Line Tools</h1>
                   </p>
 
                   <p>For other information about the command line tools, check the <a target="_blank" href="{{'cli' | helpLink}}">CLI Reference</a> and <a target="_blank" href="{{'basic_cli_operations' | helpLink}}">Basic CLI Operations</a>.</p>
+
+                  <p><b>To access the internal registry with Docker:</b>
```

Comment:
we cant have references to docker with a capital D, just say "To access the internal registry:"